dockerfiles/Dockerfile.use-installer

```diff
@@ -0,0 +1,76 @@
+ARG IMAGE="python:3-slim-buster"
+
+#---
+# Place anything that is common to both the build and execution environment in base 
+#
+FROM $IMAGE AS base
```

Comment:
`$IMAGE` is unlikely to accept very different platforms. Moreover, other stages are based on `base`. I suggest removing the envvar.